pkg/packagekit/updates.css

```diff
@@ -66,6 +66,34 @@ tr.security {
     border: none;
 }
 
+.flow-list-blank-slate {
+    padding-top: 8em;
+    margin: 0 auto;
+    max-width: 68rem;
+    text-align: center;
+}
+
+.flow-list {
+    margin: 1em 1em 3em;
+    padding: 0;
+    width: 100%;
+    max-width: 68rem;
+    text-align: left;
+    box-sizing: border-box;
+}
+
+.flow-list li {
+    text-align: left;
+    box-sizing: border-box;
+    width: 30ex;
```

Comment:
(This should make the list perfectly align @ 3 wide when constrained. Due to the above max-width, smaller browser widths should still work fine as the elements would wrap based on font-size, not percentage. This is specifically why I wanted font-based units, not percentages, here, BTW.)